test/arch.Dockerfile

```diff
@@ -1,10 +1,11 @@
 FROM archlinux
 
-RUN pacman -Sy go --noconfirm
+RUN pacman -Sy go git --noconfirm
```

Comment:
What this change is for?

Reply:
Look at line 8 in this file:
`RUN git submodule update --init`

I used git submodules in order to add [this](https://github.com/BaderBC/targz) fork of `targz` library

Reply:
I changed it, now it doesn't use git submodules